backend/src/main/java/com/praksa/auction/model/Product.java

```diff
@@ -34,7 +37,19 @@ public class Product {
     @ManyToOne
     @JoinColumn(name = "addressId")
     private Address address;
-    private Boolean shippingOption;
     private Boolean payed;
     private String phoneNumber;
+
+    public Product(NewProductDto productDto, SubcategoryService subcategoryService, PersonService personService) {
```

Comment:
Can we send subcategory value directly, without fetching it in a model ? 
Same thing with a person value ?
It is not good practice to send services to a model, then fetching something from it.